mods/ra/rules/structures.yaml

```diff
@@ -520,6 +520,9 @@ DOME:
 	InfiltrateForExploration:
 	DetectCloaked:
 		Range: 10c0
+		UpgradeTypes: powered
+		UpgradeMaxEnabledLevel: 0
+		UpgradeMinEnabledLevel: -1
```

Comment:
Since that's the case, how do I handle DetectCloaked requiring an upgrade granted from a singular source?